policy/LsstSimMapper.yaml

```diff
@@ -205,18 +199,8 @@ datasets:
     storage: YamlStorage
     tables: raw
     template: processEimage_metadata/v%(visit)d-f%(filter)s/s%(snap)i/R%(raft)s/S%(sensor)s.yaml
-  deep_measureMulti_config:
-    persistable: Config
-    python: lsst.meas.multifit.measureMulti.MeasureMultiConfig
-    storage: ConfigStorage
-    tables: raw
-    template: config/deep_measureMulti.py
   deep_safeClipAssembleCoadd_metadata:
     template: deep_safeClipAssembleCoadd_metadata/%(filter)s/%(tract)d/%(patch)s.yaml
-  deepCoadd_measMatch:
-    template: deepCoadd-results/%(filter)s/%(tract)d/%(patch)s/srcMatch-%(filter)s-%(tract)d-%(patch)s.fits
-  deepCoadd_measMatchFull:
-    template: deepCoadd-results/%(filter)s/%(tract)d/%(patch)s/srcMatchFull-%(filter)s-%(tract)d-%(patch)s.fits
   deepCoadd_forced_src:
```

Comment:
We should also remove `dcrCoadd_measMatch`, `dcrCoadd_measMatchFull`, `dcrCoadd_forced_src`, `dcrCoadd_forced_config`, and `dcrCoadd_forced_metadata`

Reply:
Done.  The package may be on its way out, but I bet it's still being copied from at times.